src/alignment/bottom.rs

```diff
@@ -178,6 +181,8 @@ mod test {
             ".#.#....##...#......###....#..",
             "..............................",
             "..............................",
+            "                              ",
+            "                              ",
```

Comment:
@rfuest Apparently, word processors (at least MS Word), when the text is bottom aligned and line height is > 1, behave like this test case and put space even after the line. TBH I don't really know what to think about this behavior as I would expect the last line to be "properly" bottom aligned. This is hinted at by the language used in Word (i.e. Line *Spacing*).

There are a few possible solutions that could be applied here:
 - leave it as is - inflexible
 - based on the vertical alignment, align the characters vertically to the line (i.e. Center would put space around the characters, Bottom would put space above the characters...) - inflexible, but might look better?
 - add a new in-line alignment option (I guess this could be similar to e-g's baseline alignment) - this would be complicated as e-t still forces Top baseline alignment - it's in a box, after all.
 - add new vertical alignments or alignment sub-types - this just sounds like a mess, but it's simple and flexible.

Do you have an opinion or suggestion here? Since we call the property "line height" instead of "line spacing" now, I guess that fits what's happening, but I'm a bit concerned as to what users of this combination actually expect.

Reply:
One advantage of consistently applying the line height/spacing to the bottom is that the vertical alignment doesn't matter if only one line of text is used. In a word processor this might be an advantage if you add text to a table, where two adjacent fields might have one or more lines and one of them is bottom aligned and the other top aligned.

But the last time I checked we don't write a word processor :wink:. Without considering any other programs I would expect vertical alignment to work as in solution 2. With solution 1 there would be no way of starting the text right at the bottom of the box, if line height > 100%. And IMO this would be a limitation for a GUI focused API.

Adding additional setting would be more flexible, but IMO this would just complicate things without a lot of benefits. IMO a padding setting for the text box would be more useful in the future, which could be used to replicate the word processor behavior in some situations.

Reply:
Thanks for your input. For now I've decided to just omit the space from the last line - which wasn't actually one of my original four points. We'll see how this turns out. I think the only percievable difference compared to 2. would be the background.